values.html

```diff
@@ -0,0 +1,188 @@
+						<p>
+							We support and encourage STEAM education where it is needed and
+							appropriate for native and indigenous communities.
+						</p>
+					</div>
+					<img
+						src="./assets/images/pages/values/mobile/education-m.jpg"
+						alt="an aboriginal woman working"
+					/>
+				</section>
+				<section class="technology values-card">
+					<div class="tech-text text-container">
+						<h2 class="h2-styles">Technology and Innovation</h2>
+						<p>
+							We advocate responsible stewardship with digital tech &
+							innovation. Technologies should not be leveraged to harm native
+							and indigenous communities or lands.
+						</p>
+					</div>
+				</section>
+			</div>
+		</div>
+		<!-- Support section -->
+		<section class="support">
+			<div class="support-header">
+				<h2 class="h2-styles">WE SUPPORT</h2>
+				<!-- add heart icon -->
+				<!-- SVG Fill value was manually changed -->
+				<svg
+					xmlns="http://www.w3.org/2000/svg"
+					width="26"
+					height="26"
+					viewBox="0 0 26 26"
+					fill="none"
+					class="heart"
+				>
+					<path
+						d="M13.0003 23.1292L11.4295 21.6992C5.85033 16.64 2.16699 13.2925 2.16699 9.20833C2.16699 5.86083 4.78866 3.25 8.12533 3.25C10.0103 3.25 11.8195 4.1275 13.0003 5.50333C14.1812 4.1275 15.9903 3.25 17.8753 3.25C21.212 3.25 23.8337 5.86083 23.8337 9.20833C23.8337 13.2925 20.1503 16.64 14.5712 21.6992L13.0003 23.1292Z"
+						fill="#b91515"
+					/>
+				</svg>
+				<!-- <img class="heart" src="./assets/icons/values/mobile/Heart-m.svg" alt="heart icon"> -->
+			</div>
+			<div class="support-text">
+				<p class="h2">
+					The United Nations Declaration on the Rights of Indigenous People.
+				</p>
+				<p>Read more about it</p>
+				<!-- </div> -->
+				<div class="support-logos">
+					<img
+						src="./assets/icons/values/mobile/globe.svg"
+						alt="globe icon"
+						class="support-logo-01"
+					/>
+					<img
+						src="./assets/icons/values/mobile/UN75_white_tagline_E 3.svg"
+						alt="UN75 icon"
+						class="support-logo-02"
+					/>
+				</div>
+			</div>
+		</section>
+		<!-- Empowering section  -->
+		<section class="empowering">
+			<div class="empowering-text">
+				<h2 class="h2-styles">Empowering Self-Determination</h2>
+				<p>
+					We advocate for native and Indigenous rights through digital
+					empowerment, aligning with the UN Declaration on the Rights of
+					Indigenous Peoples. By providing digital enablement education,
+					programming, and advisory, we support Native and Indigenous
+					communities to manage their cultural heritage and knowledge in ways
+					that meet their needs and align with their values, fostering
+					self-determination in the ever-evolving digital landscape.
+				</p>
+			</div>
+			<!-- img box -->
+			<div class="empowering-img-block">
+				<img
+					src="./assets/images/pages/values/mobile/empowering-m.jpg"
+					alt="youths working together on a project"
+				/>
+			</div>
+		</section>
```

Comment:
Can we add the footer please?

Reply:
Pulled footer and header into this branch - is that what was asked?

Reply:
Yes that's the first step. However, we also need to copy/paste the header and footer code from index.html into this file so that they appear on the page.